src/frontend/src/app.html

```diff
@@ -39,20 +39,20 @@
 
 		<!-- iOS meta tags & icons -->
 		<meta name="apple-mobile-web-app-capable" content="yes" />
-		<meta name="apple-mobile-web-app-status-bar-style" content="#33156A" />
+		<meta name="apple-mobile-web-app-status-bar-style" content="#010165" />
 		<meta name="apple-mobile-web-app-title" content="{{VITE_OISY_NAME}}" />
 		<link rel="apple-touch-icon" href="%sveltekit.assets%/favicons/apple-touch-icon.png" />
 		<link
 			rel="mask-icon"
 			href="%sveltekit.assets%/favicons/safari-pinned-tab.svg"
-			color="#33156A"
+			color="#010165"
 		/>
 
 		<!-- MS -->
-		<meta name="msapplication-TileColor" content="#33156A" />
+		<meta name="msapplication-TileColor" content="#010165" />
```

Comment:
Not in this PR but, we gonna have to generate the mstitle with a transaprent background.

<img width="235" alt="Capture d’écran 2024-07-29 à 18 50 58" src="https://github.com/user-attachments/assets/7c1e77d0-a35b-4e28-acfb-407a751d8000">

Reply:
Done with:

- PR #1913 
- PR #1914